nms/app/packages/magmalte/grafana/dashboards/CWFDashboards.js

```diff
@@ -100,8 +100,45 @@ export const CWFSubscriberDBData: GrafanaDBData = {
           description:
             "Should just be 0 for subscribers without an active session and 1 for those with. Not sure what's up with this now.",
         },
+        {
+          title: 'User Growth',
+          targets: [
+            {
+              expr: 'count(sum(session_start{msisdn=~".*"}) by (msisdn))',
+              legendFormat:
+                '',
+            },
+          ],
+          description:
+            "Number of unique subscribers over time",
+        },
+        {
+          title: 'Locally Terminated Sessions',
+          targets: [
+            {
+              expr: 'count(sum(session_manager_terminate{msisdn=~"$msisdn"}) by (msisdn))',
+              legendFormat:
+                '',
+            },
+          ],
+          description:
+            "Number of subscriber sessions terminated locally",
+        },
+        {
+          title: 'Users who used data',
```

Comment:
minor nit: User data download volume may be?

Reply:
yes. Will change to data download volume.

Reply:
Let's say "Active users" or something like that and let the description say what the metric is (I think the current description is good)

Reply:
Ok. Will make the relevant change.